RxSwiftExt.xcodeproj/project.pbxproj

```diff
@@ -1349,7 +1518,7 @@
 				);
 				INFOPLIST_FILE = "$(SRCROOT)/Source/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
Minimum development target must be above 9.0.
RxSwift 6.0 requires 9.0. 
Also Package.swift requires 9.0.